cmd/crio/main.go

```diff
@@ -80,6 +80,20 @@ func catchShutdown(ctx context.Context, cancel context.CancelFunc, gserver *grpc
 	}()
 }
 
+const usage = `OCI-based implementation of Kubernetes Container Runtime Interface Daemon
+
+crio is meant to provide an integration path between OCI conformant runtimes
+and the kubelet. Specifically, it implements the Kubelet Container Runtime
+Interface (CRI) using OCI conformant runtimes. The scope of crio is tied to the
+scope of the CRI.
+
+1. Support multiple image formats including the existing Docker image format.
```

Comment:
Can we change "Docker image format" to "Docker and OCI image formats"?

Reply:
:+1: